radio/src/opentx.cpp

```diff
@@ -1514,13 +1514,10 @@ void getADC()
 /**
  * Read ADC using 10 bits
  */
-inline uint16_t read_adc10(uint8_t adc_input) 
+uint16_t read_adc10(uint8_t adc_input) 

```

Comment:
Not related to your changes, but now you are on it ... this part of code should be in the driver! Perhaps an adc_driver.cpp with consistent function name adcRead(...)